buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -106,7 +106,7 @@ fun Project.configureIntellijPlugin() {
 
     val item = changelog.getOrNull(changelog.version.get())
     if (item != null) {
-      changeNotes.set("<h3>v${rootProject.version}</h3>\n\n" + changelog.renderItem(item, Changelog.OutputType.HTML))
+      changeNotes.set(changelog.renderItem(item, Changelog.OutputType.HTML))
```

Comment:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/3383210/215282666-1506e0d7-1697-4cfe-ab3d-99655125cb52.png">

<img width="428" alt="image" src="https://user-images.githubusercontent.com/3383210/215282733-9651569a-7ed7-4e0c-b1b7-6cc43e8c3091.png">


No longer duplicated, neither in 2022.2 (no tabs) nor in 2022.3 (change notes in a separate tab)